docs_user/modules/proc_adopting-telemetry-services.adoc

```diff
@@ -2,7 +2,13 @@
 
 = Adopting Telemetry services
 
-Adopting Telemetry means that an existing `OpenStackControlPlane` custom resource (CR), where Telemetry services are supposed to be disabled, should be patched to start the service with the configuration parameters provided by the source environment.
+The observability solution that is used in  the source environment (STF) is not the observability solution used to retrieve metrics and logs in the target environment. It has been developed with a better integration with both OpenStack and OpenShift. This means that different backends are used, rendering old data not automatically migrable.
+
+Metrics and logs are considered short-lived data and thus they are not intended to be migrated to the target environment. A procedure to adopt legacy autoscaling stack templates to the target environment is provided in the autoscaling documentation.
+
+Adopting Telemetry means that the solution will be removed from the cluster and the new solution will be deployed in the target environment, allowing for metrics (and optionally logs) retrieval and storage in the new backends.
+
+To achieve this an existing `OpenStackControlPlane` custom resource (CR), where Telemetry services are supposed to be disabled, should be patched to start the service with the configuration parameters provided by the source environment.
```

Comment:
```suggestion
The observability solution that is used in the source environment, Service Telemetry Framework, is not the observability solution that is used to retrieve metrics and logs in the target environment. The solution is developed to better integrate with both {rhos_prev_long} and {OpenShift}. You cannot automatically migrate old data because different back ends are used.

Metrics and logs are considered short-lived data and are not intended to be migrated to the target environment. A procedure to adopt legacy autoscaling stack templates to the target environment is provided in the autoscaling documentation.

If you adopt Telemetry services, the solution is removed from the cluster and the new solution is deployed in the target environment, allowing for metrics, and optionally logs, to be retrieved and stored in the new back ends.

To adopt Telemetry services, you patch an existing `OpenStackControlPlane` custom resource (CR) that has Telemetry services disabled to start the service with the configuration parameters that are provided by the source environment.
```

Reply:
@jlarriba I'm assuming that "autoscaling documentation" means this [1], but let me know if not.
[1] https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/adopting_a_red_hat_openstack_platform_17.1_deployment/adopting-openstack-control-plane-services_migrating-databases#adopting-autoscaling_adopting-bare-metal-provisioning

Reply:
No, "autoscaling documentation" is the autoscaling documentation for RHOSO, nothing to do with adoption. It has not been published yet, that's why I dont have a proper link.

Thank you for the edits.